introduction.md

```diff
@@ -3,4 +3,5 @@ Introduction
 * Name - Troy Stolte
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?